frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/ai/ai_writer_block_component.dart

```diff
@@ -475,6 +465,52 @@ class OverlayContent extends StatelessWidget {
     );
   }
 
+  List<Widget> _bottomActions(
+    BuildContext context,
+    bool showActionPopup,
+    bool hasSelection,
+    Color borderColor,
+  ) {
+    if (!showActionPopup) {
+      return [];
+    }
+
+    final document = editorState.document;
+    if (isDocumentEmptyForAI(document)) {
+      final documentContext = editorState.document.root.context;
+      if (documentContext == null) {
+        return [];
+      }
+      final view = documentContext.read<ViewBloc>().state.view;
+      if (view.name.isEmpty) {
+        return [];
+      }
+    }
+
+    return [
+      const VSpace(4.0 + 1.0),
```

Comment:
Why is it 4 + 1?

Reply:
Oh. Right. I will write a comment to explain it.

I configured the border of the main message box to be on the outside to hide some graphical artifacts. So this is is the original 4px gap + the border 1px